### PR TITLE
Implement basic trails "health" service by copy pasting code from OrgRegistry.

### PR DESCRIPTION
**Pull request by a-stacey**

### CODE REVIEW

File: projects/trails/trails.cabal [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+-- hash: 9ddefe1b8765d2a18815448a609374f23dbef6c0d6d03716dc86f19f33c474af
+
+name:           trails
+version:        0.1.0.0
+description:    Please see the README on GitHub at <https://github.com/data61/Mirza/projects/trails#readme>
+homepage:       https://github.com/data61/Mirza
+bug-reports:    https://github.com/data61/Mirza/issues
+author:         Author name here
+maintainer:     example@example.com
+copyright:      2017, 2018, 2019 CSIRO
```
[the remark this answers is not in the document]
Comment (a-stacey) [reply projects/trails/trails.cabal]: I'm not really sure what should be here, whether you just need to include the current year or whether you need to include all years and as code was copied from elsewhere in the codebase I'm not sure if its supposed to be the year that it was copied from or the year in which it was copied. I've just followed the style used elsewhere but I'm not really sure. @paulsamways thoughts?

File: projects/trails/src/Mirza/Trails/Types.hs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+instance HasKatipContext (TrailsContext) where
+  katipContexts = trailsKatipLogContexts
+  katipNamespace = trailsKatipNamespace
+
+
+-- *****************************************************************************
+-- Error Types
+-- *****************************************************************************
+
+data ORError
```
[the remark this answers is not in the document]
Comment (a-stacey) [reply projects/trails/src/Mirza/Trails/Types.hs]: Thanks for picking this up: aa03d8e2e594b2996b4a0683aa1d8481ca182748

File: projects/trails/trails.cabal [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+library
+  exposed-modules:
+      Mirza.Trails.API
+      Mirza.Trails.Client.Servant
+      Mirza.Trails.Handlers.Health
+      Mirza.Trails.Main
+      Mirza.Trails.Service
+      Mirza.Trails.Types
+  other-modules:
+      Paths_trails
```
[the remark this answers is not in the document]
Comment (a-stacey) [reply projects/trails/trails.cabal]: Have removed, but I don't understand this construct, what is it for? It just came from the template. aa03d8e2e594b2996b4a0683aa1d8481ca182748

[the remark this answers is not in the document]
Comment (sajidanower23) [reply projects/trails/trails.cabal]: I could be wrong, but I believe it is an autogenerated file (the file would be called `Paths_trails.hs` in this case). It contains information about where the data file(s) resides in your system.

It is actually not needed for any of our packages.

File: projects/trails/trails.cabal (hunk 1 above)
[the remark this answers is not in the document]
Comment (paulsamways) [reply projects/trails/trails.cabal]: It should be the year we are asserting copyright from, is this is new work, then 2019 is fine, but no need to update in future.

[the remark this answers is not in the document]
Comment (a-stacey) [reply projects/trails/trails.cabal]: Given that the oldest code in the repo seems to be 2017 based on git commit history and copyright in 97d9a915199749b6fedbbc6af5e1521321c15990 and that this was effectively created by copying code in from elsewhere in the project I assume that 2017 is then the correct year to use and have updated accordingly. aa03d8e2e594b2996b4a0683aa1d8481ca182748

**Pull request merged**